postcss.config.js

```diff
@@ -0,0 +1,11 @@
+const postcss = require('postcss');
```

Comment:
After looking at this some more I think it might be simpler to keep the `default` config able to compile the full SUIT library (with all the required plugins, like `rework-suit`) and then allow a config. I think splitting the defaults and the `postcss-config.js` is more complexity than we need. After all, this tool is not much use if it doesn't compile SUIT by default.

Also not too sure on how useful passing a config option will be, as to use additional plugins you'd need to install them in the `suitcss` directory right?